pkg/compiler/compiler.go

```diff
@@ -59,6 +59,13 @@ func (c *Compiler) CompileTests() error {
 }
 
 func (c *Compiler) compileMain(binaryName, path string) error {
+	// do not build if binary is already there
+	binPath := filepath.Join(path, binaryName)
+	if executebleExists(binPath) {
+		c.logger.Log("info", "main binary exists, not building")
```

Comment:
As we discussed in the last lunch talk, there is a new logging format which we should use in order to leverage more nifty EFK stack features.

Reply:
Great, where can I read the details?

Reply:
We still need to write it down. There have been PRs though in which we introduced this. The easiest out there is something like his: https://github.com/giantswarm/operatorkit/blob/39f1a4387b0e8ffb148f54ca8b15eee7af537ca6/framework/framework.go#L555.

Reply:
Nice, done PTAL.